R/dettl.R

```diff
@@ -15,8 +15,8 @@
 ##'
 ##' @examples
 ##' path <- dettl:::prepare_test_import(
-##'   system.file("examples", "person_information", package = "dettl"),
-##'   system.file("examples", "dettl_config.yml", package = "dettl"))
+##'   dettl:::dettl_file("examples", "person_information"),
```

Comment:
The examples should continue to use `system.file` - they should use only public API features

Reply:
I see okay, I should be exporting `prepare_test_import` then too?

Reply:
I've updated that back to using `system.file`. I've also gone through the examples and made sure I'm only using exported functions.